src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -330,5 +330,56 @@
     <bean id="batchFileLookupable" parent="batchFileLookupable-parentBean" scope="prototype"/>
     <bean id="batchFileLookupable-parentBean" class="org.kuali.kfs.kns.lookup.KualiLookupableImpl" abstract="true"
           p:lookupableHelperService-ref="batchFileLookupableHelperService"/>
+          
+    <bean id="purgeJob" parent="purgeJob-parentBean">
+        <property name="steps">
+            <list merge="true">
+                <ref bean="purgeTablesStep"/>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="purgeTablesStep" parent="purgeTablesStep-parentBean"/>
+    <bean id="purgeTablesStep-parentBean" class="edu.cornell.kfs.sys.batch.PurgeTablesStep"
+          parent="step" abstract="true" p:tablesPurgeService-ref="tablesPurgeService">
+    </bean>
+
+    <bean id="tablesPurgeService" parent="tablesPurgeService-parentBean"/>
+    <bean id="tablesPurgeService-parentBean" class="edu.cornell.kfs.sys.batch.service.impl.TablesPurgeServiceImpl"
+          abstract="true" p:tablePurgeRecordsDao-ref="tablePurgeRecordsDao">
+          <property name="tablesDetailsForPurge">
+              <list>
+                  <ref bean="concurEventNotificationTableDetailsForPurge"/>
+              </list>
+          </property>
+    </bean>
+    
+    <bean id="concurEventNotificationTableDetailsForPurge" class="edu.cornell.kfs.sys.businessobject.TableDetailsForPurge">
+        <constructor-arg type="java.lang.Class">
```

Comment:
Could these be passed in as properties rather than constructor arguments?

Reply:
I am trying to figure out how to get your suggestion to work when there needs to be an instance of edu.cornell.kfs.sys.businessobject.TableDetailsForPurge in the array tablesDetailsForPurge that is being looped through when performing the purge.
I tried using “property” in the xml file at first but the server would not load.  The errors lead me down the path to setting up the beans with “constructor” since array list elements needed to be created.